### DISCUSSION TITLE
[Enhancement]: Preview The Markdown Text

### DISCUSSION THREAD

**Discussion by ischenxin** [massCodeIO/massCode#299] [Ideas]
### Describe the bug

Not a bug, just a feature

Firstly, thank you for the author providing such a good app. It helps me a lot.
I am used to take notes with markdown, but masscode can only display the original content, I hope it can preview the text and makes content  more beautiful, eg `github` `typora`

It's possible?

The editor of the masscode is like this
<img width="1041" alt="image" src="https://user-images.githubusercontent.com/89346248/197746658-578592f0-69a7-4384-8cc1-1f968c6a097a.png">

For example, `Typora`  editor
<img width="1018" alt="image" src="https://user-images.githubusercontent.com/89346248/197746125-09de3cdf-8a75-4d73-9ca3-5a656a816dbe.png">

### To reproduce

...

### Version

Version: 2.10.0 Electron: 16.2.3 Chrome: 96.0.4664.174 Node.js: 16.9.1

### OS

macos 

### Validations

- [X] Follow our [Code of Conduct](https://github.com/massCodeIO/massCode/blob/master/CODE_OF_CONDUCT.md)
- [X] Check that there isn't [already an issue](https://github.com/massCodeIO/massCode/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/massCodeIO/massCode/discussions).

**Comment by Faq** [comment]
For me massCode shows fine (using eye icon on right top side) or I misunderstood what you meant:
Version 3.4.1, @ischenxin just update your app.
![image](https://user-images.githubusercontent.com/322016/198838906-0ad70164-b16c-4637-948d-ab848137d7e3.png)

**Reply by ischenxin** [reply]
oh, you are right, the preview function is great, thx👍

**Comment by ischenxin** [comment]
It solved my problem, The issue can be closed🙏
